lib/service/Service.ts

```diff
@@ -283,6 +287,11 @@ class Service extends EventEmitter {
         orders.sell = [...orders.sell, ...ownOrders.sell];
       }
 
+      if (!limit) {
+        // get 10 best orders
```

Comment:
The logic here per what kilian and I discussed should be `if (limit > 0)` then `slice(0, limit)`